pkg/kubelet/rkt/rkt.go

```diff
@@ -135,7 +135,9 @@ const (
 	defaultNetworkName = "rkt.kubernetes.io"
 
 	// defaultRequestTimeout is the default timeout of rkt requests.
-	defaultRequestTimeout = 2 * time.Minute
+	// Value is slightly offset from 2 minutes to make timeouts due to this
+	// constant recognizable.
+	defaultRequestTimeout = 2*time.Minute - 1*time.Second
```

Comment:
The two can't be running at the same time, so just assumed all runtime request timeouts could share this.